### ISSUE TITLE
stnbhwd package not preloaded

### ISSUE THREAD

**Issue by kepricon** [viorik/ConvLSTM#4]
Hi there

I installed stn(https://github.com/qassemoquab/stnbhwd.git) with replacement BilinearSamplerBHWD.lua with ConvLSTM/BilinearSamplerBHWD.lua
and i also installe extracunn(https://github.com/viorik/extracunn.git)

When i run the main-mnist.lua i got the error like below.
I think nn.BilinearSamplerBHWD is missing or having problem.

Can you tell me how can i solve this problem?

Best Regards,
DH

.../torch/install/share/lua/5.1/stn/BilinearSamplerBHWD.lua:1: stnbhwd package not preloaded
stack traceback:
	[C]: in function 'error'
	...ricon/Project/torch/install/share/lua/5.1/trepl/init.lua:378: in function 'require'
	main-mnist.lua:9: in main chunk
	[C]: in function 'dofile'
	...ject/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00406670

**Comment by viorik** [comment]
Hi @kepricon , 
Could you, please, try to reinstall stnbhwd package and don't replace BilinearSamplerBHWD. The current version of convLSTM is supposed to take care of that automatically, thanks to @nicholas-leonard's PR. Let me know if that works. 
Cheers.

**Comment by kepricon** [comment]
Hi @viorik 
First of all thank you for urgent reply.

I reinstalled stnbhwd without replacement and the error message(stnbhwd package not preloaded) which i reported disappeared.

However another message is shown to me like below. I've also installed several packages such as moses, rnn, dpnn, qulua, etc. though. 

I am still finding solution for this problem.
Until getting my solution, if you know the root cause of this problem.

Please let me know.

DH.

kepricon@kepricon-server:~/git/ConvLSTM$ th main-mnist.lua 
8000 20 64 64	
Loaded 8000 images	
==> training model	
Number of parameters 703651	
Number of grads 703651	
==> iteration = 1, average loss = 0.0037770224244971 lr 0.0001	
/home/kepricon/Project/torch/install/bin/luajit: ...ricon/Project/torch/install/share/lua/5.1/trepl/init.lua:378: module 'qt' not found:No LuaRocks module found for qt
	no field package.preload['qt']
	no file '/home/kepricon/.luarocks/share/lua/5.1/qt.lua'
	no file '/home/kepricon/.luarocks/share/lua/5.1/qt/init.lua'
	no file '/home/kepricon/Project/torch/install/share/lua/5.1/qt.lua'
	no file '/home/kepricon/Project/torch/install/share/lua/5.1/qt/init.lua'
	no file './qt.lua'
	no file '/home/kepricon/Project/torch/install/share/luajit-2.1.0-alpha/qt.lua'
	no file '/usr/local/share/lua/5.1/qt.lua'
	no file '/usr/local/share/lua/5.1/qt/init.lua'
	no file '/home/kepricon/.luarocks/lib/lua/5.1/qt.so'
	no file '/home/kepricon/Project/torch/install/lib/lua/5.1/qt.so'
	no file './qt.so'
	no file '/usr/local/lib/lua/5.1/qt.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'error'
	...ricon/Project/torch/install/share/lua/5.1/trepl/init.lua:378: in function 'require'
	...ricon/Project/torch/install/share/lua/5.1/image/init.lua:1327: in function 'display'
	main-mnist.lua:124: in function 'main'
	main-mnist.lua:162: in main chunk
	[C]: in function 'dofile'
	...ject/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00406670

**Comment by viorik** [comment]
Try to run it with qlua, not with th, since th does not support display as far as I know.

**Comment by kepricon** [comment]
Do i only have to run with qlua?
My environment is access to the server with terminal using ssh protocol.
As far as i know i am not able to use graphical things.

**Comment by viorik** [comment]
If you cannot use qlua, then I would recommend to put opt.display to 'false' in opts-mnist.lua (line 33). This will disable the display and it should work to run with th.

**Comment by kepricon** [comment]
Thank you Viorik.
It is working with th.

**Comment by viorik** [comment]
Could you, please, close this issue if the problem is solved? 
Cheers.
